view/frontend/web/js/script.js

```diff
@@ -0,0 +1,7 @@
+import Vue from './lib/vue.js'
+import index from './components/index.vue'
+
+var app = new Vue({
+    el: '#checkout',
```

Comment:
Is it possible to remove hardcode on element id from js files and determinate target element which was specified inside `x-magento-init`?

Reply:
That would be good

Reply:
http://devdocs.magento.com/guides/v2.1/javascript-dev-guide/javascript/js_init.html

We can get dom element and passed config.

Reply:
We are not going to support any core Magento features related to JS modules, b/c that's why we are writing it using Vue and Webpack, to make this simpler for front-ends, not follow the same overcomplicated practices from the core.